bondcpp/src/bond.cpp

```diff
@@ -165,17 +163,6 @@ Bond::~Bond()
   connectTimerCancel();
```

Comment:
(GitHub UI won't let me leave comments above here)

Two things:

1.  The comment that is above this block doesn't seem to make sense anymore, as there is no mutex in here anymore.
2. Why is deadpublishingTimerCancel() commented out?  Can we uncomment it?

Reply:
> The comment that is above this block doesn't seem to make sense anymore, as there is no mutex in here anymore.

Comment is removed.

> Why is deadpublishingTimerCancel() commented out? Can we uncomment it?

It is unclear what this is supposed to be for.  It was introduced as part of the ROS 2 port, and does not appear in the ROS 1 version at all.  As far as I can tell, it was never used and has never been uncommented.

I'm going to remove it in a separate commit that we can always come back and revert if someone feels very strongly about the feature.